### ISSUE TITLE
Error building on OS X 10.11

### ISSUE THREAD

**Issue by alexreg** [mariusae/trickle#13]
See https://gist.github.com/69c858ae2f2b34d6449eefc492a87baa for the `make` log.

What's going on here?

**Comment by pelly** [comment]
Having the same problem - @alexreg did you find a way to fix this?

**Comment by alexreg** [comment]
It seems the `make` got a bit further when I told `configure` that CC and CXX should be GCC (rather than clang), but still some errors. Looks like some Linux-specific headers are expected?

https://gist.github.com/9dd5740bf1ee1e28a43fda10e36b52ce

**Comment by pelly** [comment]
Found a different tool that satisfied my needs (limited upload speed):
http://stackoverflow.com/questions/9659382/installing-apples-network-link-conditioner-tool

**Comment by alexreg** [comment]
Yeah, I came across that too. I think the optimal way is to use `pf` to do traffic shaping, however.
